### ISSUE TITLE
Fresh Build - MYSQL issues

### ISSUE THREAD

**Issue by Zodly** [suriyun-production/mmorpg-kit-docs#540]
Couple of things I've ran into while doing a fresh build with MYSQL as the storage base.

1. When you set up everything in the build for MYSQLDatabase settings, even going as far as set all the settings inside the MYSQL Database Object, the server still requires at config file that is not created nor found within the project. You have to create it yourself.

2. The portion of code below seems to cause a loop that locks up the server from running after the config is read. If you comment it out the server starts immediately, if you dont the server just locks up with the info "Config Found"
           ```
 // Migrate data
            foreach (BuildingEntity prefab in GameInstance.BuildingEntities.Values)
            {
                ExecuteNonQuery("UPDATE buildings SET entityId=@entityId, dataId=0 WHERE dataId=@dataId",
                    new MySqlParameter("entityId", prefab.EntityId),
                    new MySqlParameter("dataId", prefab.name.GenerateHashId()));
            }
```

Unity: 2020.1.0f1
MMORPG KIT: 1.57
Additional Info: Fresh Build - No other changes but Database Options selected.

**Comment by insthync** [comment]
Oh, I will fix it.

**Comment by insthync** [comment]
Fixed in 1.57b
